contracts/src/chainlink/cairo/ocr2/IAggregator.cairo

```diff
@@ -8,6 +8,13 @@ struct Round:
     member updated_at : felt
 end
 
+struct Transmission:
```

Comment:
This is an internal struct (not part of the interface) that's why I wouldn't move it to the interface.

What actually makes sense here is to split the `aggregator.cairo` to the `Aggregator/library.cairo` lib + the contract implementation itself (uses the lib). Then the Transmission struct could be part of the library and reused by other contracts (Mock for example).

Don't do this as part of this PR. Let's just move it back for now.

Reply:
ok